domain/makefile

```diff
@@ -26,17 +26,6 @@ build:
 	cd addressassignment && go build
 	cd serviceconfigfile && go build
 
-test: build
-	go test $(GOTEST_FLAGS)
-	cd host && go test $(GOTEST_FLAGS)
-	cd pool && go test $(GOTEST_FLAGS)
-	cd servicedefinition && go test $(GOTEST_FLAGS)
-	cd servicetemplate && go test $(GOTEST_FLAGS)
-	cd service && go test $(GOTEST_FLAGS)
-	cd servicestate && go test $(GOTEST_FLAGS)
-	cd addressassignment && go test $(GOTEST_FLAGS)
-	cd serviceconfigfile && go test $(GOTEST_FLAGS)
-
 clean:
```

Comment:
Remove recursive make testing scheme in favor of go test ./... recursive testing scheme.